### PR TITLE
TCVP-1777, clean up and bug fix for staff portal

### PR DESCRIPTION
**Pull request by Sharkww**
# Description

This PR includes the following proposed change(s):

- TCVP-1777
- Citizen portal clean up
- NGRX fix for staff portal, auto-refresh after assign

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/citizen-portal/src/app/components/find-dispute/find-dispute.component.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ export class FindDisputeComponent implements OnInit {
     if (!validity) {
       return;
     }
-    this.busy = this.violationTicketService.searchTicket(this.form.value).subscribe(res => res);
```
Comment (lfdgov) [inline src/frontend/citizen-portal/src/app/components/find-dispute/find-dispute.component.ts:55]: review the warnings given below

File: src/frontend/citizen-portal/src/app/core/interceptors/backend-http.interceptor.ts [hunk 2]
```diff
@@ -62,14 +59,14 @@ export class BackendHttpInterceptor implements HttpInterceptor {
   private handleTicketsRequests(
     requestMethod: string
```
Comment (lfdgov) [inline src/frontend/citizen-portal/src/app/core/interceptors/backend-http.interceptor.ts:60]: remove commented out code?

File: src/frontend/citizen-portal/src/app/components/find-dispute/find-dispute.component.ts (hunk 1 above)
Comment (Sharkww) [reply src/frontend/citizen-portal/src/app/components/find-dispute/find-dispute.component.ts:55]: this line is just the copied line from the search ticket page, changed to search dispute

File: src/frontend/citizen-portal/src/app/core/interceptors/backend-http.interceptor.ts (hunk 2 above)
Comment (Sharkww) [reply src/frontend/citizen-portal/src/app/core/interceptors/backend-http.interceptor.ts:60]: just reverted the files

**Review by lfdgov — CHANGES_REQUESTED** [review]
remove any changes related to province & country as I am working on that
your changes have generated some warnings have a look in your PR regarding use of the store function

**Review by lfdgov — APPROVED** [review]

**Pull request merged**